src/main/java/jdbchomework/utils/InjectorJdbc.java

```diff
@@ -13,26 +13,26 @@
 import jdbchomework.dao.jdbc.DevelopersJdbcDao;
 import jdbchomework.dao.jdbc.ProjectsJdbcDao;
 import jdbchomework.dao.jdbc.SkillsJdbcDao;
-import jdbchomework.entity.Company;
-import jdbchomework.entity.Customer;
-import jdbchomework.entity.Developer;
-import jdbchomework.entity.Project;
-import jdbchomework.entity.Skill;
 import jdbchomework.service.CompanyService;
 import jdbchomework.service.CustomerService;
 import jdbchomework.service.DeveloperService;
 import jdbchomework.service.ProjectService;
 import jdbchomework.service.SkillService;
-import org.hibernate.SessionFactory;
 
-public class Injector {
+import java.sql.Connection;
 
-    private static SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
-    private static CompaniesJdbcDao companiesJdbcDao = new CompaniesJdbcDao(sessionFactory, "Company", Company.class);
-    private static CustomersJdbcDao customersJdbcDao = new CustomersJdbcDao(sessionFactory,"Customer", Customer.class);
-    private static DevelopersJdbcDao developersJdbcDao = new DevelopersJdbcDao(sessionFactory,"Developer", Developer.class);
-    private static ProjectsJdbcDao projectsJdbcDao = new ProjectsJdbcDao(sessionFactory, "Project", Project.class);
-    private static SkillsJdbcDao skillsJdbcDao = new SkillsJdbcDao(sessionFactory,"Skill", Skill.class);
+public class InjectorJdbc {
+
+    private InjectorJdbc() {
+        throw new IllegalAccessError("Utility class");
+    }
+
+    private static Connection connection = ConnectionUtil.getConnection();
+    private static CompaniesJdbcDao companiesJdbcDao = new CompaniesJdbcDao(connection, "companies", "company_id");
+    private static CustomersJdbcDao customersJdbcDao = new CustomersJdbcDao(connection, "customers", "customer_id");
+    private static DevelopersJdbcDao developersJdbcDao = new DevelopersJdbcDao(connection, "developers", "developer_id");
+    private static ProjectsJdbcDao projectsJdbcDao = new ProjectsJdbcDao(connection, "projects", "project_id");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)